### PR TITLE
M sc1 59 19

### PR DESCRIPTION
**Pull request by DrYarosh**
Please check my homework. Thank

### CODE REVIEW

**Comment by pluhin** [comment]
Conflicts!!!

File: ayarohovich/task-4-3.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html> 
+<HTML> 
+<HEAD><META charset="utf-8"><TITLE>Пример страницы</TITLE>
```
Comment (pluhin) [inline ayarohovich/task-4-3.html:3]: Where is here close part of head?

File: ayarohovich/task-4-3.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+<LINK REL="stylesheet" HREF="stylesS.css" TYPE="text/css" media="screen"> 
+<LINK REL="stylesheet" HREF="stylesP.css" TYPE="text/css" media="print">
+
+
+
```
Comment (pluhin) [inline ayarohovich/task-4-3.html]: And where is body? What is the Frankenstein here?

File: ayarohovich/task-4-4.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+<HTML> 
+<HEAD><META charset="utf-8"><TITLE>Пример страницы</TITLE>
+<style>
+	
+	/* Вариант 4 */
+	@import "stylesS.css" screen;
+	@import "stylesP.css" print;
+	/* Вариант 4 */
+</style>
+
```
Comment (pluhin) [inline ayarohovich/task-4-4.html]: Frankenstein

File: ayarohovich/task-4-5.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+<HTML> 
+<HEAD><META charset="utf-8"><TITLE>Пример страницы</TITLE>
+<style>
+	
+	@import "stylesM.css";
+	
+	
+</style>
+
+
```
Comment (pluhin) [inline ayarohovich/task-4-5.html]: Frankenstein :)

**Review by pluhin — CHANGES_REQUESTED** [review]

**Review by pluhin — APPROVED** [review]
Good, approved

**Comment by DrYarosh** [comment]
Thаnks

ср, 23 янв. 2019 г., 17:47 Siarhei Pishchyk <notifications@github.com>:

**Pull request merged**